onnxruntime/python/tools/quantization/CalTableFlatBuffers/KeyValue.py

```diff
@@ -35,18 +38,19 @@ def Value(self):
             return self._tab.String(o + self._tab.Pos)
         return None
 
-
+def Start(builder): builder.StartObject(2)
 def KeyValueStart(builder):
-    builder.StartObject(2)
-
-
+    """This method is deprecated. Please switch to Start."""
```

Comment:
sounds like we will need to move away from the deprecated api's ? (KeyValueAddKey, KeyValueAddValue etc.)
if not in this PR, then let's add an item to clean it up later

Reply:
This file is automatically generated by flatbuffers compiler with the schema:
https://github.com/microsoft/onnxruntime-inference-examples/blob/main/quantization/object_detection/trt/yolov3/trt_cal_table.fbs
I'm still figuring out how to generate the include files without deprecated messages.

Reply:
understood. 
I just noticed we are still using the deprecated api's in quant_utils.py is all.

Reply:
Also, from flatbuffers's example code, it has the similar deprecated messages
https://github.com/google/flatbuffers/blob/master/tests/MyGame/Example2/Monster.py#L21

Reply:
yes, it's good they give us heads up on what api's to use/not use.

Reply:
> understood. I just noticed we are still using the deprecated api's in quant_utils.py is all.

Yes, I noticed as well. We should refactor the code in another PR.